console_libraries/menu.lib

```diff
@@ -61,6 +61,19 @@
 {{ template "_menuItem" (args . "cassandra.html" "Cassandra") }}
 {{ end }}
 
+{{ if query "up{job=~'blackbox.*'}" }}
```

Comment:
What you're looking for is specific to your deployment, what we provide be default needs to work reasonably for everyone out of the box. The consoles we provide need to work off one job, like the others. If you want to customise that further then you should do that locally.